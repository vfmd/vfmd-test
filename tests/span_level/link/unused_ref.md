A ref definition is not part of the output,
whether [used] or not.

[used]: url
[not used]: url
