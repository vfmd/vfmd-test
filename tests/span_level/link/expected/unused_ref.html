<p>A ref definition is not part of the output,
whether <a href="url">used</a> or not.</p>
